plug-ins/groups/group_combat.cpp

```diff
@@ -151,30 +168,26 @@ VOID_SPELL(ChainLightning)::run( Character *ch, Character *victim, int sn, int l
 SPELL_DECL(ChillTouch);
 VOID_SPELL(ChillTouch)::run( Character *ch, Character *victim, int sn, int level ) 
 { 
-    
-    Affect af;
     int dam;
+    
+    act_p("Прикосновение ледяной длани $c2 замораживает $C4!",
+           ch,0,victim,TO_NOTVICT,POS_RESTING);
+    act_p("Прикосновение твоей ледяной длани замораживает $C4!",
+           ch,0,victim,TO_CHAR,POS_RESTING);
+    act_p("Прикосновение ледяной длани $c2 замораживает тебя!",
+           ch,0,victim,TO_VICT,POS_RESTING);
 
-    dam = number_range(1,level);
-    if ( !saves_spell( level, victim,DAM_COLD,ch, DAMF_SPELL ) )
-    {
-        act_p("$c1 замерзает от ледяного прикосновения.",
-               victim,0,0,TO_ROOM,POS_RESTING);
-        af.where     = TO_AFFECTS;
-        af.type      = sn;
-        af.level     = level;
-        af.duration  = 6;
-        af.location  = APPLY_STR;
-        af.modifier  = -1;
-        af.bitvector = 0;
-        affect_join( victim, &af );
-    }
-    else
-    {
+    // Tier 2 damage (spell has regular effects)
+         if (level <= 20)   dam = dice( level, 8  );    
+    else if (level <= 40)   dam = dice( level, 12 );
+    else if (level <= 70)   dam = dice( level, 15 );
+    else                    dam = dice( level, 18 );
+    
+    if ( saves_spell( level, victim,DAM_COLD,ch, DAMF_SPELL ) )
         dam /= 2;
-    }
 
     damage_nocatch( ch, victim, dam, sn, DAM_COLD,true, DAMF_SPELL );
+    cold_effect(victim,level,dam,TARGET_CHAR, DAMF_SPELL);    
```

Comment:
@fiorine just fyi, cold_effect has a very weird saves check that depends on damage. For spell level 100, you only need to save against level 72. For spell level 50, it's enough to save against level 32 and so on. So this PR effectively reduces chill effect of the spell.